airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/BigQueryUploaderFactory.java

```diff
@@ -141,7 +163,13 @@ private static BigQueryDirectUploader getBigQueryDirectUploader(
         .setProject(bigQuery.getOptions().getProjectId())
         .build();
 
-    final TableDataWriteChannel writer = bigQuery.writer(job, writeChannelConfiguration);
+    final TableDataWriteChannel writer;
+
+    try{
```

Comment:
nit: space between `try` and `{`

Reply:
Updated, thanks. Please ignore the code-style issues for now. Once everything is approved I usually run the gradle's "format" command along with version bumping. So such issues would be automatically fixed.